v2.1/debug-zip.md

```diff
@@ -7,13 +7,18 @@ toc: true
 The `debug zip` [command](cockroach-commands.html) connects to your cluster and gathers the following information from each active node into a single file (inactive nodes are not included):
 
 - [Log files](debug-and-error-logs.html)
+- [Cluster events](monitoring-and-alerting.html#events-to-alert-on)
 - Schema change events
 - Node liveness
 - Gossip data
 - Stack traces
 - Range lists
 - A list of databases and tables
-- Heap profiles (**new in v2.0**)
+- [Cluster Settings](cluster-settings.html)
+- [Metrics](admin-ui-custom-chart-debug-page.html#available-metrics)
+- Alerts
```

Comment:
How do "Alerts" and "Cluster events" differ?

Reply:
As I understand it, Alerts = the configured alerts (https://www.cockroachlabs.com/docs/dev/monitoring-and-alerting.html#events-to-alert-on), while Cluster Events = events  like node joining the cluster, decommissioning, etc.

@bdarnell Please correct me if I am wrong.

Reply:
Yeah, that's the right idea, although the alerts collected here are not exactly the same as those on the monitoring and alerting page. Alerts are things like "3 under-replicated ranges" or "raft group has no leader" that probably represent a problem with the cluster (the kind of thing we'd show in a red box in the web UI). Right now there's not much here but we plan to add to it. 

Cluster events, on the other hand, are "normal" things like node starts and stops, table creations, etc.

Reply:
OK, in that case, let's remove the cluster events link to the monitoring docs. That's misleading. Fine to just leave it a bullet.

Reply:
Fixed.